packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3179,6 +3164,28 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// Whether decedents of this RenderObject are allowed to add semantic
+  /// information to the [SemanticsNode] annotated by this widget.
+  ///
+  /// When set to false decedents are allowed to annotate [SemanticNode]s of
+  /// their parent with the semantic information they want to contribute to the
+  /// semantic tree.
+  /// When set to true the only way for decedents to contribute semantic
```

Comment:
if you were hoping for a paragraph break here you won't get one (if you weren't, it's fine)